index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Mobile Version Completion</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div id="main-div">
+        <header>
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+              <svg viewBox="0 0 100 80" width="40" height="40" class="svg-menu">
+                <rect y="38" width="100" height="8" rx="8" fill="#6070FF"></rect>
+                <rect y="58" width="100" height="8" rx="8" fill="#6070FF"></rect>
+                <rect y="78" width="100" height="8" rx="8" fill="#6070FF"></rect>
+              </svg>
+            </nav>
+        </header>
+        <section id="introduction-paragraphs">
+            <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
+            <p class="description">I'm a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <label class="connect-message">LET'S CONNECT</label>
+
+            <div id="link-media">
+                <ul class="social-icons">
```

Comment:
- [Optional] Kindly consider using an indentation of two spaces in all your documents. THis is a convention and also best practice.